### PR TITLE
<weather-viewer>

### PR DESCRIPTION
**Pull request by mrbernnz**

### CODE REVIEW

**Comment by mrbernnz** [comment]
@snuggs 
screenshot 2017-05-14 23 23 51

**Comment by snuggs** [comment]
@mrbernnz Oddly fixed this probably at the same time you made the PR. From 
 2 days ago.


https://github.com/devpunks/snuggsi/commit/8ec00e49f4d375e01af04949ae6a1e056835784a

**Comment by snuggs** [comment]
@mrbernnz rebasin atop master. This branch is a couple hundred commits behind.

**Comment by snuggs** [comment]
@mrbernnz used the following as a reference for geolocation api. I think a good stick figure for our happy path. Also the root of the site is great for Mobile UX / Desktop for some modern APIs. Go through them. Takes about 5 min to demo them all. @brandondees i think we are getting close to having a similar demo. Like a 1 through 10 easy peasy examples.

https://mobilehtml5.org
Geolocation API example -> https://mobilehtml5.org/ts/?id=4

**Comment by mrbernnz** [comment]
@snuggs How do we send the actual temperature object (line 50)?  Presuming it's coming from a server.

**Comment by snuggs** [comment]
@mrbernnz how about we use https://openweathermap.org/current#geo

Here is an example of the JSON payload we have access to for token assignment. 

http://samples.openweathermap.org/data/2.5/weather?lat=35&lon=139&appid=b1b15e88fa797225412429c1c50c122a1


capture d ecran 2017-05-18 a 01 32 49

**Comment by snuggs** [comment]
@brandondees ever work with this api? ☝️

**Comment by brandondees** [comment]
@snuggs nope but it looks pretty simple. i'd suggest trying to keep the data binding decoupled from the data source at some level. not necessarily a whole separate adapter component or anything but just minimize the number of places you'll have code to update if you want to pull similar info in a totally different structure from some alternative source. all that said, it's just nice-to have YAGNI stuff until you actually have a use case for a secondary source of data.

**Comment by snuggs** [comment]
@mrbernnz ping

**Comment by snuggs** [comment]
capture d ecran 2017-05-20 a 14 26 01

**Comment by snuggs** [comment]
@brandondees be careful what you ask for... /cc @mrbernnz 

https://github.com/devpunks/snuggsi/blob/17108eb1dab50ed58ea2833ce94f05cc9466925f/examples/weather-viewer/weatherman.js

File: examples/weather-viewer/weatherman.js [hunk 1]
```diff
@@ -1 +1,28 @@
...
+
+    return fetch (api)
+      .then (response => response.json ())
+  }
+
+  static get api () {
+  }
+
+  get key () // from openweathermap.org
+    { return '62b74592e7bde09ede21693bce86460a' }
```
Comment (mrbernnz) [inline examples/weather-viewer/weatherman.js]: @snuggs Is this ok?

**Pull request merged**